File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int size=nums.length;
        int[] ans=new int[2];
    for(int i=0;i<size;i++){
       int num1=nums[i];
       for(int j=i+1;j<size;j++){
        if(num1+nums[j]==target){
            ans[0]=i;
            ans[1]=j;
           return ans;
        }
       }
       
    }
    return ans;
    

    }
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        
     unordered_map<int,int>mp;
     for(int num:nums){
        mp[num]++;
     }

     vector<int>ans;
     int criteria=nums.size()/3;
     for(auto it:mp){
        if(it.second>criteria)
        ans.push_back(it.first);
     }
     return ans;

    }
};